internal/app.go

```diff
@@ -182,6 +182,12 @@ var (
 		EnvVars: []string{"TRIVY_IGNORE_POLICY"},
 	}
 
+	listAllPackages = cli.BoolFlag{
+		Name:    "list-all-pkgs",
+		Usage:   "enabling the option will output all packages regardless of vulnerability",
+		EnvVars: []string{"TRIVY_LIST_ALL_PACKAGES"},
```

Comment:
```suggestion
		EnvVars: []string{"TRIVY_LIST_ALL_PKGS"},
```
Hi. I am no expert in Golang, and hence, I am not sure if I need to provide a similar suggestion elsewhere in the PR.

I have just noticed this and wanted to give you heads-up for "consistency" reasons. A similar CLI switch is `--removed-pkgs` with the corresponding environment variable `TRIVY_REMOVED_PKGS`. Perhaps, you would like this one to be shorter, too.

Reply:
Changes pushed, thanks for suggestion